alacritty/src/display.rs

```diff
@@ -20,6 +20,10 @@ use std::fmt::{self, Formatter};
 use std::sync::atomic::Ordering;
 use std::time::Instant;
 
+#[cfg(target_os = "macos")]
+use cocoa::base::{id, nil, NO};
+use cocoa::foundation::{NSString, NSUserDefaults};
```

Comment:
Attributes on Rust only refer to the next line, so you need to add this attribute twice.

```suggestion
#[cfg(target_os = "macos")]
use cocoa::base::{id, nil, NO};
#[cfg(target_os = "macos")]
use cocoa::foundation::{NSString, NSUserDefaults};
```